core/main/src/broker/endpoint_broker.rs

```diff
@@ -901,53 +901,43 @@ pub trait EndpointBroker {
     /// Generic method which takes the given parameters from RPC request and adds rules using rule engine
     fn apply_request_rule(rpc_request: &BrokerRequest) -> Result<Value, RippleError> {
         if let Ok(mut params) = serde_json::from_str::<Vec<Value>>(&rpc_request.rpc.params_json) {
-            if params.len() > 1 {
```

Comment:
There was a bug here where if there were no params to passed into the request, then we wouldn't process the Request transform at all. @satlead please confirm I understand your original intention WRT the request params.

Reply:
These changes preserve the original logic of expecting `last` to be the second request parameter, if it exists at all.